build_tools/third_party/tracy_client/CMakeLists.txt

```diff
@@ -4,6 +4,13 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+# Traced builds require the headers.
+set(IREE_HDRS_ROOT_PATH "${IREE_ROOT_DIR}/third_party/tracy/public")
```

Comment:
we could fix this by duplicating some of the tracy C API - wouldn't be a lot of work and would let us hide this if you think it's worth it

Reply:
Yeah, I had a similar thought but fine for the moment. I have more work to do on the bundled deps stuff and this patch is a big enough lift as it is.